eng/common/templates/job/performance.yml

```diff
@@ -7,7 +7,7 @@ parameters:
   container: ''                   # required -- name of the container
   osGroup: ''                     # required -- operating system for the job
   extraSetupParameters: ''        # optional -- extra arguments to pass to the setup script
-  frameworks: ['netcoreapp3.0']   # optional -- list of frameworks to run against
+  frameworks: ['netcoreapp3.1']   # optional -- list of frameworks to run against
```

Comment:
⚠️ This is arcade code /cc @tmat

Reply:
Well ... I probably shouldn't change that then.